cockatrice/src/player.cpp

```diff
@@ -2347,6 +2351,31 @@ void Player::actIncPT(int deltaP, int deltaT)
     game->sendGameCommand(prepareGameCommand(commandList), playerid);
 }
 
+void Player::actResetPT()
+{
+    int playerid = id;
+    QList<const ::google::protobuf::Message *> commandList;
+    QListIterator<QGraphicsItem *> selected(scene()->selectedItems());
+    while (selected.hasNext()) {
+        CardItem *card = static_cast<CardItem *>(selected.next());
+        CardInfoPtr info = card->getInfo();
+        Command_SetCardAttr *cmd = new Command_SetCardAttr;
+        QString zoneName = card->getZone()->getName();
+        cmd->set_zone(zoneName.toStdString());
+        cmd->set_card_id(card->getId());
+        cmd->set_attribute(AttrPT);
+        QString ptString = info->getPowTough();
+        cmd->set_attr_value(ptString.toStdString());
+        commandList.append(cmd);
+
+        if (local) {
```

Comment:
You set playerid at the top for no apparent reason to possibly modify it here. A suggestion could be to ternary case it, or if/else

Reply:
Also, you're checking if it's a local game each time... can probably do it just once

Reply:
I could but then I'd have to change that everywhere else too.

Reply:
Ohh.. hmm. Refactor at some point will be necessary, but this is good to go :)

Reply:
Is there a ticket for that refactor with more details?